web/src/containers/activity/EntryDetails.test.js

```diff
@@ -19,8 +19,6 @@ jest.mock('react-router-dom', () => {
   };
 });
 
-global.scrollTo = jest.fn();
```

Comment:
We shouldn't have to call 'scrollTo' anywhere in the app. This is handled by `<AutoScrollToElement/>`.